src/application.d.ts

```diff
@@ -115,6 +115,11 @@ declare namespace Application {
      * politeness level
      */
     announce?: String | RouteAnnounce
+    /**
+     * Register hooks for the route
+     */
+    hooks?: object
+
```

Comment:
we should probably specify what hooks are available, and the signature of those hooks (currently only before)